README.md

```diff
@@ -270,6 +270,10 @@ int main()
 
 # Verifying the library
 
+## Clang Tidy
+
+If `clang-tidy` is installed, then CMake will create a `make tidy` target that can be used to run `clang-tidy` over the codebase. The configuration for the checkers can be found in the `.clang-tidy` file in the root of the repository.
```

Comment:
The target is not called `make tidy`. The target is called `tidy`.

Reply:
Fixed